content/working_with/console/widgets/deploymentsView.md

```diff
@@ -0,0 +1,279 @@
+---
+layout: bt_wiki
+title: Deployments View
+category: Widgets
+draft: false
+---
+
+Presents information about deployments in a single view. Contains:
+
+- a table showing the deployments in the system (the left side) The deployments
+  can be further filtered.
+  See [Filtering deployments]({{< relref "#filtering-deployments" >}}).
+- a pane with the deployment details divided into tabs (the right side)
+- a map showing the deployments locations
+
+<!-- TODO: screenshot -->
+
+## Table
+
+The table shows deployments that match the current filter.
+See [Filtering deployments]({{< relref "#filtering-deployments" >}})
+for more details on filtering.
+
+The columns shown in the table can be shown and hidden using the widget's
+configuration.
+
+<!--  TODO: screenshot -->
+
+The columns can show:
+
+1. The deployment status represented as an icon
+
+   - No icon - _good_
+   - Yellow spinner - _in progress_
+   - Red exclamation mark - _requires attention_
+
+   See [the Environments and Services section]({{< relref "#environments-and-services" >}})
+   for more information on deployment statuses.
+
+2. ID - the deployment ID
+
+   You can see the exact ID by hovering over the _ID_ button. From the popup
+   you can also copy the ID directly to your clipboard.
+
+3. Name - the deployment display name
+4. Blueprint Name - the name of the blueprint that this deployment is using
+5. Environment Type - the value of the `csys-env-type` label for that
+   deployment
+
+   Useful for differentiating deployments that use the same blueprint.
+
+6. Location - the name of the site the deployment is assigned to
+7. The counts and statuses of child deployments depending on their type
```

Comment:
In the https://docs.cloudify.co/staging/RD-1529-documentation-for-Deployments-View/working_with/console/widgets/deploymentsview/ the list numbers are not the same as here:
![image](https://user-images.githubusercontent.com/5202105/118935406-cd1f5300-b94b-11eb-8439-c3929201c1ac.png)

Reply:
Good point :+1: I'll try to fix it by indenting the nested paragraphs

Reply:
Looks like it was the indentation indeed. It works with 4 spaces:

![image](https://user-images.githubusercontent.com/889383/118972853-5006d480-b971-11eb-9052-07538b189510.png)

https://docs.cloudify.co/staging/RD-1529-documentation-for-Deployments-View/working_with/console/widgets/deploymentsview/

Reply:
Great 👍🏻